tests/pyGeometry_test.py

```diff
@@ -38,7 +38,7 @@ def test_verticesnber_candler(self):
         m = pyprt.ModelGenerator([shape_geo_from_obj])
         model = m.generate_model([attrs], rpk, 'com.esri.pyprt.PyEncoder', {
                                  'emitReport': False, 'emitGeometry': True})
-        self.assertEqual(len(model[0].get_vertices()), 97050*3)
+        self.assertEqual(len(model[0].get_vertices()), 97044*3)

```

Comment:
PRT 2.4 more aggressively merges duplicate vertices (independent of the encoder "vertex merge" option). this behavior is reproducible within CityEngine as well -> no problem of this PR.